src/test/java/org.spongepowered.mod/event/SpongeModEventManagerTest.java

```diff
@@ -75,6 +96,7 @@ public void testSpawn() throws ClassNotFoundException {
 
         eventBus.register(forgeListener);
 
+        this.eventManager.registerListeners(this.plugin, listener);
```

Comment:
I don't think this is supposed to be here. This section of the test is making sure that the fields are `true` when only Forge listeners are registered.

Reply:
Original MR message stated

    Also adds another EventManager#registerListeners() call in testSpawn(), 
    as this should be expected behavior (we don't want to fire an event if
    nothing is listening for it - wasted processing time)."

You could be right, though. Could we have someone else's thoughts?